docs/ubuntu_install.rst

```diff
@@ -57,18 +52,16 @@ following output in your terminal.
 Create the Ubuntu installation media
 ------------------------------------
 
-To create the Ubuntu installation media, you can either burn the ISO
-image to a CD-R or create a bootable USB stick (see instructions for
-doing this on `OS
-X <http://www.ubuntu.com/download/desktop/create-a-usb-stick-on-mac-osx>`__,
-`Ubuntu <http://www.ubuntu.com/download/desktop/create-a-usb-stick-on-ubuntu>`__
-and
-`Windows <http://www.ubuntu.com/download/desktop/create-a-usb-stick-on-windows>`__).
-As a reliable method we recommend using the ``dd`` command to copy the
-hybrid ISO directly to a USB drive rather than a utility like UNetbootin
-which can result in errors. Once you have a CD or USB with an ISO image
-of Ubuntu on it, you may begin the Ubuntu installation on both
-SecureDrop servers.
+To create the Ubuntu installation media, you can either burn the ISO image to a
+CD-R or create a bootable USB stick.  As a reliable method we recommend using
+the ``dd`` command to copy the hybrid ISO directly to a USB drive rather than a
+utility like UNetbootin which can result in errors. Once you have a CD or USB
+with an ISO image of Ubuntu on it, you may begin the Ubuntu installation on both
+SecureDrop servers. If your USB is mapped to /dev/sdb, you would use dd like so:
+
+::
```

Comment:
Fixed in https://github.com/fowlslegs/securedrop/commit/caca462e2b3fb764587d7d42c3e6d0971fb80981